sdk/eventhub/azure-eventhub/stress/values.yaml

```diff
@@ -0,0 +1,13 @@
+# Optional list of scenarios. If specified multiple stress test jobs will be generated,
+# one for each scenario in the list. The pod spec can then be configured to pass the
+# scenario name down to the test command, e.g. `command: ["node", "{{ .Scenario }}.js"]`
```

Comment:
for maintainability, would you be able to update the README file with instructions on how to specify:
- run one scenario locally
- run multiple scenarios locally

Would you be able to clarify what "configuring the pod spec" means? (I'm not super familiar with all the k8 setup stuff 😅.) And is it configured to running all of these scenarios by default now? If not, maybe in the README, you could also add a section on how to modify this pod spec to update the scenarios/whatever instructions make sense? (Just so, someone completely new would be able to read it and figure out how to run multiple scenarios + deploy with multiple scenarios. (And maybe even linking the stress docs in there could be helpful?)

Reply:
Yes! I have plans to make a new ReadME on the stress/ folder level for what each test is doing and how the setup works

Reply:
amazing, thank you!